joy: {
  assign_x: 0,
  assign_y: 1,
  assign_z: 3,
  assign_x_straight: 6,
  assign_y_straight: 7,

  max_x: -0.05,
  max_y: 0.05,
  max_z: 0.01,
  max_x_straight: -0.05,
  max_straight: -0.05,
}

twoWD: {
  first_x: 0,
  first_y: 0,

  radius: 0.1,
  tread: 0.3,
  enc_cpr: 4096,
}

arduino: {
  # PIDの順
  gainL: [13.0, 1.0, 1.0],
  gainR: [13.0, 1.0, 1.0],
}